bhohnco/kingsCastle
# Feature/main page build layout

Is this a fix or a feature?
* feature

What is the change or fix?
*  Cards have book cover photos 
* Cards in a grid
* Book titles are showing in cards 

How should this be tested?
* load the page in the local browser 
* check for the favorite button 

Linked Issues: #9 #10 
Add A Reviewer if Necessary
*
